08-rails/preview-rails.md

```diff
@@ -0,0 +1,43 @@
+# Preview of Rails
+
+## Introduction
+
+As we transition from exploring programming fundamentals and making CLI programs into web development, we will need to shift our focus.
+
+## Learning Goals
+
+At the end of this lesson, you should be able to
+
+1. adjust expectations for what kinds of problems we will be solving
+1. have resources to follow for preparing your computer (development environment) for Ruby on Rails development
+1. have resources to follow for preparing your perspective for Ruby on Rails development
+
+<!-- ## Examples of Rails apps -->
+<!-- Maybe add some visuals about different Rails apps? The auditorium lecture for Rails day 1 already includes some of this, though -->
```

Comment:
I don't know if you imagined screenshots or links here but the [Rails website](rubyonrails.org) lists these:
- Basecamp
- GitHub
- Shopify
- Airbnb
- Twitch
- SoundCloud
- Hulu
- Zendesk
- Square
- Cookpad